smart-contract-sale/Makefile

```diff
@@ -0,0 +1,37 @@
+.PHONY: all
```

Comment:
You can also group all `.PHONY` targets together as well: `.PHONY: fmt, build, ...`. I'm not sure if one style if preferred over another.

Reply:
gotcha; no idea what style is preferrable; this is copied/pasted from existing SC setups